### ISSUE TITLE
Show/Hide Shortcut not surfaced in Chrome

### ISSUE THREAD

**Issue by skube** [GoogleChromeLabs/ProjectVisBug#556]
By default, one can <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>D</kbd> to show/hide **VizBug**. This shortcut can be been be seen while hovering over the extension icon (i.e. the tooltip).

However, when using Google Chrome and modifying the shortcut via `chrome://extensions/shortcuts` the tooltip doesn't update to reflect the current shortcut. 

It's a little thing—I'm not even sure if it's possible to fix in this project. ❤️

**Comment by argyleink** [comment]
oh interesting! let me forward this on to some folks 👍🏻

**Comment by dotproto** [comment]
Yeah, you can use [`chrome.commands.getAll()`](https://developer.chrome.com/docs/extensions/reference/commands/#method-getAll) ([MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/commands/getAll)) to retireve an array of commands that the extension has registered. If there's a collision with a command's shortcut, the `shortcut` property for that command will be an empty string.
